src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+---
+title: Responsive Web Design Basics
+subhead: How to create sites which respond to the needs and capabilities of the device they are viewed on.
+description: |
+  How to create sites which respond to the needs and capabilities of the device they are viewed on.
+date: 2019-02-12
+updated: 2020-05-14
+authors:
+ - petelepage
+ - rachelandrew
+tags:
+ - blog
+ - css
+ - layout
+ - mobile
+ - ux
+draft: true
+---
+
+* [Set the viewport](#viewport)
+* [Size content to the viewport](#size-content)
+* [Use CSS media queries for responsiveness](#media-queries)
+* [How to choose breakpoints](#breakpoints)
+* [View media query breakpoints in Chrome DevTools](#devtools)
+
+The use of mobile devices to surf the web continues to grow at an astronomical pace, and these devices are often constrained by display size and require a different approach to how content is laid out on the screen.
+
+Responsive web design, originally defined by [Ethan Marcotte in A List Apart](http://alistapart.com/article/responsive-web-design/), responds to the needs of the users and the devices they're using. The layout changes based on the size and capabilities of the device. For example, on a phone users would see content shown in a single column view; a tablet might show the same content in two columns.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="resize.webm" type="video/webm; codecs=vp8">
+    <source src="resize.mp4" type="video/mp4; codecs=h264">
```

Comment:
```suggestion
    <source src="https://storage.googleapis.com/web-dev-assets/responsive-web-design-basics/resize.mp4" type="video/mp4">
```

* For some reason the webm file is larger than the mp4, whereas usually it's much smaller. Did you convert this or was this already in the google/webfundamentals repo? We can just omit it for now.
* We have a storage bucket where we host videos. We may someday also host images on the bucket (or provide some other GUI).
* Somebody just reported a bug related to the `codecs=vp8` and `codecs=h264` directives so we will no longer be specifying those in our video markup.

Reply:
These were the files from the existing article, I've not changed them.